Resources/Prototypes/Entities/Mobs/NPCs/carp.yml

```diff
@@ -88,6 +88,43 @@
   - type: TypingIndicator
     proto: guardian
 
+- type: entity
+  name: Lia
+  parent: MobCarpMagic
+  id: MobCarpLia
+  description: A failed experiment of Nanotrasen to create weaponised carp technology. This less than intimidating carp now serves as the Head of Security's pet.
+  components:
+    - type: AiFactionTag
+      factions:
+        - Nanotrasen
+    - type: InteractionPopup
+      interactSuccessString: petting-success-carp
+      interactFailureString: petting-failure-generic
+
+- type: entity
+  name: Cayenne
+  parent: MobCarpMagic
+  id: MobCarpCayenne
+  description: A failed Syndicate experiment in weaponized space carp technology, it now serves as a lovable mascot.
+  components:
+    - type: AiFactionTag
```

Comment:
yea this carp is gonna beat the shit out of nukies. Someone should probably fix factions so that syndies and nukies have the proper faction tag.